llvm/tools/llvm-reduce/deltas/ReduceOperandsToArgs.cpp

```diff
@@ -108,7 +108,7 @@ static void replaceFunctionCalls(Function *OldF, Function *NewF) {
     NewCI->setCallingConv(NewF->getCallingConv());
     NewCI->setAttributes(CI->getAttributes());
 
-    if (auto *FPOp = dyn_cast<FPMathOperator>(NewCI))
+    if (isa<FPMathOperator>(NewCI))
       NewCI->setFastMathFlags(CI->getFastMathFlags());
```

Comment:
Use FPOp instead of NewCI

Reply:
fixed

Reply:
'setFastMathFlags' is a private member of 'llvm::FPMathOperator'